### PR TITLE
Ref T40666, Fix publish script 

### PR DESCRIPTION
**Pull request by luis30n**

### CODE REVIEW

File: scripts/publish.sh [hunk 1]
```diff
@@ -1,21 +1,13 @@
 #!/bin/bash
 set -e
 
-# Install and publish
-if git describe --exact-match >&/dev/null; then
```
Comment (Angelmmiguel) [inline scripts/publish.sh:5]: @luis30n please keep this conditional as it only executes the publish process when there's a new tag associated to the commit.

File: scripts/publish.sh [hunk 2]
```diff
@@ -1,21 +1,13 @@
...
-  # Install
-  yarn install
-  # Build
-  lerna bootstrap
-  lerna run build
-  # Run JS dev
-  lerna run --scope @bitnami/hex-js build:dev
-
-  # Prepare the folder
-  BUCKET=$BUCKET yarn run cdn
```
Comment (Angelmmiguel) [inline scripts/publish.sh:18]: This step is required. This task is the one that pushes new artifacts to S3

**Review by Angelmmiguel — CHANGES_REQUESTED** [review]
I added some comments.

File: scripts/publish.sh (hunk 1 above)
Comment (luis30n) [reply scripts/publish.sh]: Sorry I moved it to draft due to that, I removed the conditional for testing.

File: scripts/publish.sh (hunk 2 above)
Comment (luis30n) [reply scripts/publish.sh]: Same for this, that's why I moved the PR to draft

**Review by Angelmmiguel — APPROVED** [review]
LGTM!

**Pull request merged**